install/environment.yml

```diff
@@ -1,16 +1,9 @@
 name: zarp
 channels:
+  - bioconda
   - conda-forge
   - defaults
 dependencies:
-  - graphviz=2.40.1
-  - jinja2=3.0.3
   - mamba=0.16.0
-  - networkx=2.4
-  - pip=20.0.2
-  - pygments=2.6.1
-  - pygraphviz=1.3
   - python=3.7.4
-  - pip:
-    - pandas==1.0.1
-    - snakemake==7.2.1
+  - snakemake=7.2.1
```

Comment:
Just a question: why did we decide to drop _pip_ and installing everything from _conda_? I guess its cleaner that way but was there any reason we had it differently?

Reply:
I just tried the as little as possible approach. Since it is working with less lines i went for it